cmake/VulkanCodegen.cmake

```diff
@@ -9,6 +9,9 @@ set(VULKAN_GEN_ARG_ENV "")
 if(USE_VULKAN_RELAXED_PRECISION)
   string(APPEND VULKAN_GEN_ARG_ENV "precision=mediump")
 endif()
+if (USE_VULKAN_FP16_INFERENCE)
+  string(APPEND VULKAN_GEN_ARG_ENV "format=rgba16f")
```

Comment:
Should we update BUCK file, too?

Reply:
yep, this will be updated in a separate diff.